controllers/front/ExpressCheckout.php

```diff
@@ -293,4 +306,22 @@ private function addressAlreadyExist($alias, $id_customer)
 
         return (int) Db::getInstance()->getValue($query);
     }
+
+    /**
+     * @param string $state
+     *
+     * @return int|null
+     */
+    private function getStateId($state)
```

Comment:
I would move this function to CountryReporisoty leave as little code in controller as possible

Reply:
Done